dashboard/config/levels/custom/spritelab/sl_level_0419.level

```diff
@@ -0,0 +1,717 @@
+<GamelabJr>
```

Comment:
should this be here?

Reply:
Definitely not! This is just a local file I made for testing functions without the modal. Deleted from branch!